docs/components/tasklist/userguide/tasklist-example-use-case.md

```diff
@@ -0,0 +1,77 @@
+---
+id: tasklist-example-use-case
```

Comment:
I almost feel like this is more of "Getting started with Tasklist" or even just "Getting started" in the sidebar -- I think an example use case is almost like a particular task tutorial that may belong in guides. WDYT?